### PR TITLE
CORS addition to my service

### PR DESCRIPTION
**Pull request by kimberlyjk**
added CORS to my server and service such that when proxies are trying to utilize service, the proxy is able to get in.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "babel-jest": "^26.6.3",
     "babel-preset-react": "^6.24.1",
     "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
```
Comment (hannahmanfredi) [inline package.json:24]: looking forward to learning more about this fancy cors pckg

File: server/index.js [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
 app.listen(port, () => {
   console.log(`Server listening at http://127.0.0.1:${port}`);
 });
+
+const allowedOrigins = [
+  'http://localhost:3000',
+  'http://localhost:5500',
+];
+app.use(cors());
+app.use(cors({ origin: allowedOrigins }));
```
Comment (hannahmanfredi) [inline server/index.js:21]: Oh right on, so you're adding enabling your Proxy to access every other service via this origins array. That's super cool.

File: server/index.js [hunk 3]
```diff
@@ -70,10 +82,14 @@ app.get('/overview/all', async (req, res) => {
   await axios.get('http://localhost:3001/reviews', { params: { campId } })
     .then(response => {
       // console.log('Review API Call response ', response.data);
-      data.header = { percentRec: response.data.recommendedPer}
+      console.log('Accessed Review Service!!');
```
Comment (hannahmanfredi) [inline server/index.js:85]: Great success mssg!

File: server/index.js [hunk 4]
```diff
@@ -70,10 +82,14 @@ app.get('/overview/all', async (req, res) => {
...
+      data.header = { percentRec: response.data.recommendedPer };
     })
     .catch(error => {
-      console.log('Error Ocurred ', error);
+      console.log('Unable to Access Review Service...');
```
Comment (hannahmanfredi) [inline server/index.js:89]: benevolent error handling

File: client/overview.jsx [hunk 5]
```diff
@@ -29,7 +29,7 @@ class Overview extends React.Component {
   }
 
   fetchOverview() {
-    return axios.get('/overview/all', { params: { campId: this.state.campId } })
+    return axios.get('http://localhost:3003/overview/all', { params: { campId: this.state.campId } })
```
Comment (hannahmanfredi) [inline client/overview.jsx:32]: Oh neat, is this your request to your proxy?

**Review by hannahmanfredi — COMMENTED** [review]
Hi, great job adding the CORS package and utilizing it on your front-end where you send an axios GET to your proxy. IThe code all looks great. I tried but wasn't able to run your proxy and test the new CORS setup.

File: client/overview.jsx (hunk 5 above)
Comment (kimberlyjk) [reply client/overview.jsx:32]: no it's just a request directly to my service. I'm not sure if it's the right thing to do, but once I got my proxy running I saw that when i made the simple `/overview/all` request it was trying to make it on the proxy's url and I wanted to make it of my service url.

**Pull request merged**